scripts/gen/windows-resources

```diff
@@ -15,30 +16,28 @@ trap 'rm -rf $TEMPDIR' EXIT
 
 if [ "$(go env GOHOSTOS)" == "windows" ]; then
 	WINDRES=windres
-	WINDMC=windmc
```

Comment:
Sounds right. I copied these from `moby/moby` which used it for building the daemon binary as well. I guess I missed this line while I was pruning the file.